pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/fraser/omstillingsstoenad/OmstillingsstoenadFellesFraser.kt

```diff
@@ -122,10 +130,10 @@ object OmstillingsstoenadFellesFraser {
                             "+47 ${Constants.KONTAKTTELEFON_PENSJON}, kvardagar 09:00–15:00. Det vil gjere det " +
                             "enklare for oss å gi deg rask og god hjelp om du oppgir fødselsnummer.",
                     English to "You can find answers to your questions online: ${Constants.OMS_URL}. " +
-                            "Feel free to chat with us or write to us here: ${Constants.KONTAKT_URL}. " +
-                            "You can also contact us by phone (+47 ${Constants.KONTAKTTELEFON_PENSJON}), " +
-                            "weekdays 09:00-15:00. If you provide your national identity number, we can more " +
-                            "easily provide you with quick and good help."
+                            "Feel free to chat with us or write to us here: ${Constants.KONTAKT_URL}. You can also contact " +
+                            "us by phone (+47 ${Constants.KONTAKTTELEFON_PENSJON}), weekdays 09:00-15:00. If you " +
```

Comment:
Eventuelt Constants.Utland.KONTAKTTELEFON_PENSJON

Reply:
De har +47 på alle språkene her. Ser ut til at dem har gått for det generelt på hele OMS.